src/clr-base/utils/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "ngPackage": {
+    "lib": {
+      "entryFile": "index.ts"
+    }
```

Comment:
Utils has its own entry file now. This will be needed for importing shared code between components when added in the future. Examples:
 
https://github.com/ng-packagr/ng-packagr/pull/685
https://github.com/coryrylan/lithium/tree/master/projects/lithium

Reply:
👍